config/jobs/kubernetes-sigs/kro/kro-presubmits-main.yaml

```diff
@@ -25,3 +25,78 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cloud-provider-kro-presubmits
       testgrid-tab-name: presubmits-unit-tests
+  - name: presubmits-integration-tests
+    always_run: true
+    optional: false
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    path_alias: "sigs.k8s.io/kro"
+    cluster: eks-prow-build-cluster
+    branches:
+    - ^main$
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20251021-e2c2c9806f-1.33
+        command:
+        - scripts/ci/presubmits/integration-tests
```

Comment:
@jakobmoellerdev happy to make all of these scripts one, or just use makefile rule. But also happy to go with this and rework it later on.